wgpu-types/src/lib.rs

```diff
@@ -1177,116 +1179,123 @@ impl TextureFormat {
         let Linear = false;
         let Srgb = true;
 
-        let (features, sample_type, srgb, block_dimensions, block_size) = match self {
+        // Flags
```

Comment:
perhaps we could link to https://gpuweb.github.io/gpuweb/#texture-format-caps in a comment?